.github/workflows/ci.yml

```diff
@@ -46,10 +49,6 @@ jobs:
             calculate-code-coverage: 'yes'
             phpunit-flags: '--testsuite coverage --exclude-group covers-nothing --coverage-clover build/logs/clover.xml'
 
-          - operating-system: 'ubuntu-20.04'
-            php-version: '8.1'
```

Comment:
should we add 8.2 snapshot, or not yet available?

Reply:
Yes, for Github Action `shivammathur/setup-php@v2` 8.2 is available as nightly (and will automatically transition to stable once released): https://github.com/marketplace/actions/setup-php-action#tada-php-support

Including 8.1 in our matrix did work very well last month's PHP release.

Reply:
https://github.com/marketplace/actions/setup-php-action#nightly-build-setup

seems there is 8.2/nightly support, let me try, also nice to see the tool listed there as an example :)